packages/pricing/integration-tests/__tests__/services/rule-type/index.spec.ts

```diff
@@ -251,5 +251,24 @@ describe("RuleType Service", () => {
         })
       )
     })
+
+    it("should throw an error when using one of the reserved keywords", async () => {
+      let error
+
+      try {
+        await service.create([
+          {
+            name: "Test Rule",
+            rule_attribute: "currency_code",
+          },
+        ])
+      } catch (e) {
+        error = e
+      }
+
+      expect(error.message).toEqual(
+        "Can't create rule_attribute with reserved keywords - currency_code"
```

Comment:
suggestion: should we also provides what are the reserved keywords? something like
`"Can't create rule_attribute with one of the reserved keywords [X, Y, Z] - currency_code"`